### ISSUE TITLE
avoid `postfixOps` in code generator

### ISSUE THREAD

**Issue by xuwei-k** [scalikejdbc/scalikejdbc#1234]
```
[error] -- Error: /tmp/sbt_e60b88db/src/test/scala/app/models/ProgrammersSpec.scala:41:14 
[info] [error] 41 |      created should not beNull
[info] [error]    |      ^^^^^^^^^^^^^^^^^^
[info] [error]    |postfix operator `beNull` needs to be enabled
[info] [error]    |by making the implicit value scala.language.postfixOps visible.
[info] [error]    |----
[info] [error]    |This can be achieved by adding the import clause 'import scala.language.postfixOps'
[info] [error]    |or by setting the compiler option -language:postfixOps.
[info] [error]    |See the Scaladoc for value scala.language.postfixOps for a discussion
[info] [error]    |why the feature needs to be explicitly enabled.
```

https://github.com/scalikejdbc/scalikejdbc/blob/64d995f0a4f80f07b4adeb33e2a667ecb510a4db/scalikejdbc-mapper-generator-core/src/main/scala/scalikejdbc/mapper/CodeGenerator.scala#L917

**Comment by seratch** [comment]
Thanks for reporting this. This looks like an easy fix. If someone is interested in working on it within a few days, we are happy to have the fix in v4.0.0. Otherwise, we will have this in v4.1.

**Comment by RhnSharma** [comment]
I would like to take this one.
